src/handlers/handleSubmit.ts

```diff
@@ -0,0 +1,51 @@
+import { addTodo, USER_ID } from '../api/todos';
+import { Errors } from '../types/ErrorsEnum';
+import { Todo } from '../types/Todo';
+import { handleError } from './handleError';
+
+export const handleSubmit = (
+  inputChange: string,
+  setErrorNotification: React.Dispatch<React.SetStateAction<Errors>>,
+  setTempTodo: React.Dispatch<React.SetStateAction<Todo | null>>,
+  setIsBeingSaved: React.Dispatch<React.SetStateAction<boolean>>,
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>,
+  onInputChange: React.Dispatch<React.SetStateAction<string>>,
+  setFocusOnChange: React.Dispatch<React.SetStateAction<boolean>>,
+) => {
+  if (!inputChange) {
+    handleError(Errors.EMPTY, setErrorNotification);
+
+    return;
+  } else {
```

Comment:
Do we need an `else` block when we already exit the function when an error occurs?

Reply:
> Do we need an `else` block when we already exit the function when an error occurs?

Good point